app/assets/stylesheets/nav.scss

```diff
@@ -9,31 +9,59 @@ $button-color: #1E90FF;
   margin-left: 5%;
   list-style-type: none;
 }
-
+.top-bar {
+  background-color: #3f3f3f;
+}
+h3{
+  font-family: $font-sans;
+}
+h2{
+  color: white;
+}
+body{
+  background-color: #e9e9e9;
+  font-family: $font-sans;
+}
+body.home{
+  background-size: relative;
+  background-image: url("http://jarques.me/img/potd_sept_30.jpg");
+}
 .container {
   margin-left: 10%;
   margin-right: 5%;
-
 }
 
 .inline {
   display: inline-block;
-
   input {
     width:280px;
+    border-radius: 5%;
   }
   label {
       color: orange;
       font-weight: bold;
+      span{
+        color: gray;
+        font-size: 10px;
+      }
     }
-
 }
-
+.block {
+  display: inline-block;
+}
 .form {
   margin-left: 10%;
-
+  display: inline-block;
+}
+.centered_bar {
+  margin-top: 10em;
+  margin-left: 10%;
+}
+.home_text{
+  text-align: center;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.